tests/test_x509_adapter.py

```diff
@@ -3,14 +3,20 @@
 import unittest
 import pytest
 
-from OpenSSL.crypto import load_pkcs12
+try:
+    from OpenSSL.crypto import load_pkcs12
+except ImportError:
+    OPENSSL_AVAILABLE = False
+else:
+    OPENSSL_AVAILABLE = True
+    from requests_toolbelt.adapters.x509 import X509Adapter
```

Comment:
Why's this moved in here? Seems like it should've been in the main block and not inside a conditional

Reply:
The import fails if `pyopenssl` isn't present. I agree it's not lovely. I'm open to alternatives.

Reply:
Ah interesting! I agree then that probably means the X509Adapter should be present here in the compatibility module.